### PR TITLE
Prepare 1.1.6 release

### PR DESCRIPTION
**Pull request by SeanNaren**
## What does this PR do?

Sets up 1.1.6 Release.

## Before submitting
- [x] Was this discussed/approved via a GitHub issue? (not for typos and docs)
- [x] Did you read the [contributor guideline](https://github.com/PyTorchLightning/pytorch-lightning/blob/master/.github/CONTRIBUTING.md), **Pull Request** section?
- [x] Did you make sure your PR does only one thing, instead of bundling different changes together?
- [x] Did you make sure to update the documentation with your changes? (if necessary)
- [x] Did you write any new necessary tests? (not for typos and docs)
- [x] Did you verify new and existing tests pass locally with your changes?
- [x] Did you update the [CHANGELOG](https://github.com/PyTorchLightning/pytorch-lightning/blob/master/CHANGELOG.md)? (not for typos, docs, test updates, or internal minor changes/refactorings)

## PR review
Anyone in the community is free to review the PR once the tests have passed.
Before you start reviewing make sure you have read [Review guidelines](https://github.com/PyTorchLightning/pytorch-lightning/wiki/Review-guidelines). In short, see the following bullet-list:

 - [x] Is this pull request ready for review? (if not, please submit in draft mode)
 - [x] Check that all items from **Before submitting** are resolved
 - [x] Make sure the title is self-explanatory and the description concisely explains the PR
 - [x] Add labels and milestones (and optionally projects) to the PR so it can be classified
 - [x] **Check that target branch and milestone match!**

## Did you have fun?
Make sure you had fun coding 🙃

### CODE REVIEW

**Review by tchaton — APPROVED** [review]
LGTM !

File: CHANGELOG.md [hunk 1]
```diff
@@ -5,38 +5,24 @@ All notable changes to this project will be documented in this file.
...
-
 - Fixed FileNotFoundError for best checkpoint when using DDP with Hydra ([#5629](https://github.com/PyTorchLightning/pytorch-lightning/pull/5629))
-
-
 - Fixed an error when logging a progress bar metric with a reserved name ([#5620](https://github.com/PyTorchLightning/pytorch-lightning/pull/5620))
-
-
 - Fixed `Metric`'s `state_dict` not included when child modules ([#5614](https://github.com/PyTorchLightning/pytorch-lightning/pull/5614))
-
+- Fixed Neptune logger creating multiple experiments when gpus > 1 ([#3256](https://github.com/PyTorchLightning/pytorch-lightning/pull/3256))
```
Comment (Borda) [inline CHANGELOG.md:22]:
```suggestion
- Fixed Neptune logger creating multiple experiments when GPUs > 1 ([#3256](https://github.com/PyTorchLightning/pytorch-lightning/pull/3256)) 
```

File: CHANGELOG.md [hunk 2]
```diff
@@ -5,38 +5,24 @@ All notable changes to this project will be documented in this file.
...
-
 - Fixed an error when logging a progress bar metric with a reserved name ([#5620](https://github.com/PyTorchLightning/pytorch-lightning/pull/5620))
-
-
 - Fixed `Metric`'s `state_dict` not included when child modules ([#5614](https://github.com/PyTorchLightning/pytorch-lightning/pull/5614))
-
+- Fixed Neptune logger creating multiple experiments when gpus > 1 ([#3256](https://github.com/PyTorchLightning/pytorch-lightning/pull/3256)) 
+- Fixed duplicate logs appearing in console when using the python logging module ([#5509](https://github.com/PyTorchLightning/pytorch-lightning/pull/5509))
+- Fixed tensor printing in `trainer.test()` ([#5138](https://github.com/PyTorchLightning/pytorch-lightning/pull/5138))
+- Fixed not using dataloader when hparams present ([#4559](https://github.com/PyTorchLightning/pytorch-lightning/pull/4559))
```
Comment (Borda) [inline CHANGELOG.md:25]:
```suggestion
- Fixed not using dataloader when `hparams` present ([#4559](https://github.com/PyTorchLightning/pytorch-lightning/pull/4559))
```

**Review by Borda — APPROVED** [review]

File: CHANGELOG.md [hunk 3]
```diff
@@ -5,38 +5,24 @@ All notable changes to this project will be documented in this file.
...
-### Deprecated
-
-
-### Removed
-
+- Pass batch outputs to on_train_batch_end instead of epoch end outputs ([#4369](https://github.com/PyTorchLightning/pytorch-lightning/pull/4369))
 
 ### Fixed
 
 - Fixed `toggle_optimizer` to reset `requieres_grad` state  ([#5574](https://github.com/PyTorchLightning/pytorch-lightning/pull/5574))
```
Comment (kaushikb11) [inline CHANGELOG.md:18]: Not a change in this PR but a minor typo `requires_grad`

**Review by kaushikb11 — APPROVED** [review]

**Review by ydcjeff — APPROVED** [review]
